arlo-client/src/components/AuditFlow/BoardTable.tsx

```diff
@@ -137,20 +138,18 @@ const BoardTable: React.FC<IProps> = ({
       <RightWrapper>
         {roundComplete ? (
           <Button intent="primary">Review Complete - Finish Round</Button>
-        ) : (
-          ballots &&
-          unauditedBallot && (
-            <Link
-              to={
-                url +
-                `/round/1/batch/${unauditedBallot.batch.id}/ballot/${unauditedBallot.position}`
-              }
-              className="bp3-button bp3-intent-primary"
-            >
-              Start Auditing
-            </Link>
-          )
-        )}
+        ) : ballots.length && unauditedBallot ? (
```

Comment:
```suggestion
        ) : ballots.length > 0 && unauditedBallot && (
```
This allows you to remove the `: null` bit at the end, I believe.

Reply:
Nice touch!